lib/loaders/babel.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
                     ['@babel/preset-env', {
                         // modules don't need to be transformed - webpack will parse
                         // the modules for us. This is a performance improvement
-                        // https://github.com/babel/babel/tree/master/packages/babel-preset-env#modules
+                        // https://babeljs.io/docs/en/next/babel-preset-env.html#modules
```

Comment:
Shouldn't it be https://babeljs.io/docs/en/babel-preset-env#modules instead since we use `^7.0.0`?

Reply:
Yup...just a drive by pull request :)